src/views/Challenge.vue

```diff
@@ -176,7 +186,17 @@ export default {
               sorted.reverse()
             }
             return sorted
-          } else {
+          } else if(this.sortBy.v === 'ada_votes'){
+              let sorted = proposals.sort((a, b) => {
+              var resultA = a.upvotes - a.downvotes
+              var resultB = b.upvotes - b.downvotes
+              return (resultA < resultB) ? -1 : (resultA > resultB) ? 1 : 0
+            })
+            if (this.sortBy.r) {
+              sorted.reverse()
+            }
+            return sorted
```

Comment:
There is no need to introduce this part.
In the data structure there is already the property `votes_result` and using it will allow to skip the creation of this new case using the default one to compare numbers.
It will also solve the current inversion in the sorting order (ASC acts as DESC and vice-versa)